index.src.html

```diff
@@ -513,20 +514,24 @@ <h3 id="fetch-integration">Fetch Integration</h3>
 
   ISSUE: Monkey patching! Talk with Anne.
 
-  If the <a>`Clear-Site-Data`</a> header is present in an HTTP
-  <a>response</a>, then data MUST be cleared before rendering the response to
-  the user. That is, before step #9 in the current <a>main fetch</a> algorithm,
-  execute the following step:
+  If the <a>`Clear-Site-Data`</a> header is present in an HTTP <a>response</a>
+  received from the network, then data MUST be cleared before rendering the
```

Comment:
Done.

I copied over and slightly adapted the text from the other pull request which I closed. Let me know what you think.

By the way, should we also elaborate on cache? The fact that this is network-only means that the standard HTTP cache is also ignored (and I don't mean just service workers' caches.match() ).